engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/views/product-repositories.html

```diff
@@ -88,9 +88,12 @@
           <td bst-table-cell>{{ repository.content_type }}</td>
           <td bst-table-cell>
             <span ng-show="repository.url">
-              <span ng-show="repository.last_sync == null" translate>
+              <span ng-show="repository.last_sync == null && repository.last_sync_words == null" translate>
                 Not Synced
               </span>
+              <span ng-show="repository.last_sync == null && repository.last_sync_words" translate>
+                Completed {{ repository.last_sync_words }} ago
+              </span>
```

Comment:
@jlsherrill what do you think about putting the logic in these two spans in the controller?

Reply:
I would have to look at it more, I only suggested it as it seems like a lot of logic is going on in this view and there is some room for improvement. Maybe it would be too complicated to do for that cell since there are various structures for each option

Reply:
thanks yeah, i initially agreed, but when i went to make the change it seemed more awkward as to not being worth it.

Reply:
seems fine as is then, I approved :)